app/main.py

```diff
@@ -8,12 +8,16 @@
 import uvicorn
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
-from fastapi.responses import ORJSONResponse
+from fastapi.openapi.docs import get_redoc_html, get_swagger_ui_html
+from fastapi.responses import ORJSONResponse, RedirectResponse
 
 from .api import utility as util
 from .api.routers import attributes, query
 
-app = FastAPI(default_response_class=ORJSONResponse)
+app = FastAPI(
+    default_response_class=ORJSONResponse, docs_url=None, redoc_url=None
+)
+favicon_url = "https://raw.githubusercontent.com/neurobagel/documentation/main/docs/imgs/logo/neurobagel_favicon.png"
```

Comment:
Out of curiosity: would it make sense to just keep a copy of the favicon png in this repo in case the documentation repo [gets reorganized in the future](https://github.com/neurobagel/documentation/issues/99) and the logo file moves?

Reply:
Yea I was thought about that too but the downside there is if the logo changes we have to change the logo here as well. I guess either way if there is a change this repo has to be updated.